core/server/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -2486,10 +2486,16 @@ public FileSystemCommand workerHeartbeat(long workerId, List<Long> persistedFile
     boolean ownerGroupChanged = false;
     boolean permissionChanged = false;
     if (options.getOwner() != null) {
+      if (options.getOwner().isEmpty()) {
```

Comment:
is it possible for a v1.3 master with security = SIMPLE to get here from a replayed journal from 1.2 where NO_SASL is used but